exercism/diamond/diamond.js

```diff
@@ -0,0 +1,49 @@
+var Diamond = function() {};
+
+Diamond.prototype.makeDiamond = function(input) {
+  let letter = input.toUpperCase()
+let alphabet = "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
+let diamondLimit = []
+let diamondArray = []
+
+for (let i = 0; i < alphabet.length; i++) {
+  diamondLimit.push(alphabet[i])
+  if (alphabet[i] === letter) {
+    break;
+  }
+}
+
+let diamondSize = diamondLimit.length * 2 - 1
+
+for (let i = 0; i < diamondSize; i++) {
+  diamondArray.push(Array(diamondSize).fill(' '))
+}
+let leftLetterPlacement = diamondLimit.length - 1
+let rightLetterPlacement = diamondLimit.length - 1
+
+let i = 0
+let j = 0
+while (i < diamondArray.length/2) {
+  diamondArray[i][rightLetterPlacement] = diamondLimit[j]
+  diamondArray[i][leftLetterPlacement] = diamondLimit[j]
+  rightLetterPlacement--
+  leftLetterPlacement++
+  j++
+  i++
```

Comment:
`j` and `i` are the same in the scope of this while loop, no need to have two separate variables.